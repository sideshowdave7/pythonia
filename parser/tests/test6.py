if x not in z: print("Why?")

