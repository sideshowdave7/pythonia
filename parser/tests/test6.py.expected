(program
 (cond ((comparison x ("not-in" z)) (expr (indexed print (called "Why?"))))))

